core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -2946,14 +2952,15 @@ private void getExpressionConditionsAsList(Condition condition, List<ExpressionC
      * get the mutual members' out of it as a DN list.
      * If it's memberOf group filtering, directly get the user name list from search result.
      *
-     * @param answer
-     * @param returnedAttributes
-     * @return
+     * @param answer                     Answer
+     * @param returnedAttributes         Returned Attributes
+     * @param sizeOfExpressionConditions How many sub parts does Expression Condition has (equal to 1 for a single
```

Comment:
Shall we describe more why we added this param?

Reply:
Fixed: b4a1c359b2cdcd987db1766b9580db2a011d7f28